dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -182,15 +183,15 @@ private void CompositionHost_InvalidateDrawing(object sender, InvalidateDrawingE
             var height = e.Height;
 
             // Clear render target backbround
-            windowRenderTarget.BeginDraw();
-            windowRenderTarget.Clear(white);
-            windowRenderTarget.EndDraw();
+            _windowRenderTarget.BeginDraw();
+            _windowRenderTarget.Clear(_white);
```

Comment:
If the intention is to clear it to white, use Windows.UI.Colors.White rather than some local definition of white. 
But if your intention is to have the background color configurable then don't call it _white because it could be a different color when you change the configuration.

Reply:
updated